### ISSUE TITLE
Exit codes from scriptcs.exe are not propagated

### ISSUE THREAD

**Issue by adamralph** [scriptcs-contrib/svm#67]
It seems as though svm always returns a zero exit code, even if scriptcs.exe returns something else.

This tripped me up to today whilst writing a script pack test automation app.

**Comment by paulbouwer** [comment]
Fixed in ebd9aecd30edd66d5920a5234d5f398d50e7e9b9. It was only an issue in the `Windows` version, but has also been tested on `Linux`.

```batch
c:\ > scriptcs a.csx
ERROR: Could not find file 'C:\a.csx'. - 'C:\a.csx'.

c:\ > echo %errorlevel%
1

c:\> scriptcs -v
               _       _
 ___  ___ _ __(_)_ __ | |__ ___ ___
/ __|/ __| '__| | '_ \| __// __/ __|
\__ \ (__| |  | | |_) | |_| (__\__ \
|___/\___|_|  |_| .__/ \__\\___|___/
                |_| Version: 0.14.0

c:\ > echo %errorlevel%
0
```

```bash
paul@ubuntu:~$ scriptcs a.csx
ERROR: Could not find file "/home/paul/a.csx". - '/home/paul/a.csx'.

paul@ubuntu:~$ echo $?
1

paul@ubuntu:~$ scriptcs -v
               _       _
 ___  ___ _ __(_)_ __ | |__ ___ ___
/ __|/ __| '__| | '_ \| __// __/ __|
\__ \ (__| |  | | |_) | |_| (__\__ \
|___/\___|_|  |_| .__/ \__\\___|___/
                |_| Version: 0.14.0

paul@ubuntu:~$ echo $?
0
```

**Comment by adamralph** [comment]
:+1: great work @paulbouwer 

I also retried my use case and 0.4.1 fixed it for me.

**Comment by adamralph** [comment]
That error message from scriptcs.exe is, er, interesting https://github.com/scriptcs/scriptcs/issues/1010

**Comment by paulbouwer** [comment]
Yes - it is ... confusing :smile:

**Comment by adamralph** [comment]
Should be fixed in 0.15
